libs/portals/admin/ids-admin/src/shared/components/ContentCard.tsx

```diff
@@ -57,25 +124,108 @@ const ContentCard: FC<ContentCardProps> = ({
+                <DropdownMenu
+                  title="Sync"
+                  icon="chevronDown"
+                  menuClassName={styles.menu}
+                  items={[
+                    {
+                      title: '',
+                      render: () => (
+                        <>
+                          <Box
+                            justifyContent="center"
+                            alignItems="center"
+                            display="flex"
+                            columnGap={1}
+                            className={styles.menuItem}
+                          >
+                            <Icon
+                              icon={inSync ? 'checkmark' : 'warning'}
+                              color={inSync ? 'blue400' : 'red400'}
+                              size="small"
+                              type="outline"
+                            />
+                            <Text variant="small" color="blue400">
+                              {inSync
+                                ? formatMessage(m.inSyncAcrossAllEnvironments)
+                                : formatMessage(
+                                    m.notInSyncAcrossAllEnvironments,
+                                  )}
+                            </Text>
+                          </Box>
+                          <Divider />
+                        </>
+                      ),
+                    },
+                    ...(inSync || dirty
+                      ? []
+                      : [
+                          {
+                            title: '',
+                            render: () => (
+                              <Box
+                                display="flex"
+                                justifyContent="center"
+                                padding={2}
+                              >
+                                {checkIfLoadingForIntent() ? (
+                                  <LoadingDots large />
+                                ) : (
+                                  <button
+                                    className={styles.syncButton}
+                                    type="submit"
+                                    value={`${intent}-sync`}
+                                    name="intent"
+                                  >
+                                    <Text
+                                      variant="small"
+                                      fontWeight="semiBold"
+                                      color={'blue400'}
+                                    >
+                                      {formatMessage(m.syncSettings)}
+                                    </Text>
+                                  </button>
+                                )}
+                              </Box>
+                            ),
+                          },
+                        ]),
+                  ]}
+                  key="sync-environment"
```

Comment:
Why are you giving it an explicit key?

Reply:
Thats a good question, the keys should have been on the items and not the DropDowMenu itself

Reply:
I was getting a error message in the browser (Each child in a list should have a unique "key" prop.
Check the render method of `DropdownMenu`. ) thats why I added this key.